versions.yaml

```diff
@@ -0,0 +1,33 @@
+---
+# Sources for a base cloud os images
+cloudimg:
+  ubuntu:
+    focal:
+      amd64:
+        url: https://cloud-images.ubuntu.com/releases/focal/release-20230107/ubuntu-20.04-server-cloudimg-amd64.img
+        sha256sum: 3895e38566e5c2c019f5c6f825ab7570ee34dac6b9142fab0c7e5a78084c4280
+      s390x:
+        url: https://cloud-images.ubuntu.com/releases/focal/release-20230107/ubuntu-20.04-server-cloudimg-s390x.img
+        sha256sum: 24673aa86785573d3a92e15166ff81beff88cbb0abc01938f156eb1332e87cd3
+  centos:
+    stream-8:
+      amd64:
```

Comment:
Hi @jtumber-ibm , first of all, thanks for working on that! I honestly don't like much the versions.yaml even on kata containers but I don't have a better solution :(

Regarding this line in specific, could you change from amd64 to x86_64? Usually using "amd64" implies that we will need a function/code to translate arch (as in `uname -m`).

Reply:
Hi @wainersm, I wish everyone would just pick one and stick with it, i.e. in Go-land it's `amd64` and rust went with `x86_64` and you rightly say that the kernel reports `x86_64`. 
As this is primarily a Go project I'd sway towards using `amd64` and we already have logic to switch from `x86_64` in place. 
Happy to switch if it would make other parts of the project easier